src/core/annotation.js

```diff
@@ -814,7 +820,7 @@ class ChoiceWidgetAnnotation extends WidgetAnnotation {
     // inherit the options from a parent annotation (issue 8094).
     this.data.options = [];
 
-    let options = Util.getInheritableProperty(params.dict, 'Opt');
+    let options = getInheritableProperty({ dict, key: 'Opt', getArray: true, });
```

Comment:
`getArray` shouldn't be necessary here, since we're already iterating through *and* de-referencing the array below at https://github.com/mozilla/pdf.js/pull/9509/files#diff-18ca06e2bd5be4c9132b51f78388b8f0R827.

The only effect of using `getArray` here is that we're now having to iterate through the array *twice*, once in `Dict.getArray` and then once below.

Reply:
Good catch! I've removed that and updated the commit/pull request message.